platform/android/display_server_android.h

```diff
@@ -63,6 +63,11 @@ class DisplayServerAndroid : public DisplayServer {
 private:
 	String rendering_driver;
 
+	bool alt_mem;
+	bool shift_mem;
+	bool control_mem;
+	bool meta_mem;
```

Comment:
You can do the initialization here instead of the constructor.

Reply:
yeah... most of the code is copied from display_server_windows so I just copied what it does there